Integrations/GoogleCloudStorage/GoogleCloudStorage.py

```diff
@@ -0,0 +1,566 @@
+import traceback
+
+import demistomock as demisto
```

Comment:
@panro please remove these 3 lines

Reply:
I'm using traceback... will move it under ''' IMPORTS '''.